examples/usecases/ecommerce-session-based-next-item-prediction-for-fashion.ipynb

```diff
@@ -99,14 +99,21 @@
      "name": "stderr",
```

Comment:
<p>Is there anything blocking us for only using the V2-API in this notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/849/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>

Reply:
I will update to V2 api :) and we'll see.